stripe-core/src/main/java/com/stripe/android/core/utils/FeatureFlags.kt

```diff
@@ -4,9 +4,7 @@ import androidx.annotation.RestrictTo
 import com.stripe.android.core.BuildConfig
 
 @RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
-object FeatureFlags {
-    val customerSheetACHv2 = FeatureFlag()
-}
+object FeatureFlags
```

Comment:
Should we remove local feature flags?

Reply:
I'd vote no, but at the very least do it as a separate PR if we do.